### ISSUE TITLE
Performance GTEx model

### ISSUE THREAD

**Issue by tabeariepe** [Illumina/SpliceAI#89]
Hi, 

I tried to reproduce your results with tensorflow 2.1.0 and the additional batch normalisation layer. To test the code I only trained one model for each dataset and then tested it with the test dataset. For the canonical dataset I got a similar performance as reported in the paper:

**Acceptor**
0.9920	0.9283	0.9789	0.9857	0.9641	0.9936	0.2071	0.0015	0.0003	14289

**Donor**
0.9938	0.9334	0.9845	0.9894	0.9695	0.9945	0.2385	0.0013	0.0003	14289

 For the GTEx model, the performance is lower:

**Acceptor**
0.9924	0.7464	0.8701	0.9408	0.8299	0.9645	0.1969	0.0260	0.0042	22349

**Donor**
0.9914	0.7561	0.8838	0.9511	0.8415	0.9718	0.1953	0.0231	0.0034	22004

I could not find any information about the performance of your GTEx model. So I wonder if this is because I use a different tensorflow version or if you also observed this behaviour for the GTEx model. 

Thanks for your help in advance.

**Comment by kishorejaganathan** [comment]
These numbers are along the expected lines. The GTEx dataset is quite noisy due to the fact that the additional splice sites are not always used. If you take the model trained on the GTEx dataset and test it using the canonical dataset, you should get 90+% accuracies.

**Comment by tabeariepe** [comment]
Thank you for the response, I indeed get about 90% accuracy when I test the model trained on the GTEx dataset on the canonical dataset.
